get_azure_blob.go

```diff
@@ -0,0 +1,210 @@
+package getter
+
+import (
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	storage "github.com/Azure/azure-storage-go"
```

Comment:
Looks like this should be github.com/Azure/azure-storage-blob-go?

Reply:
Working on it.

Reply:
This has been switched over.